api/src/mailbox.rs

```diff
@@ -566,6 +579,33 @@ impl Response for StashMeasurementResp {}
 // No command-specific input args
 // No command-specific output args
 
+// CERTIFY_KEY_EXTENDED
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct CertifyKeyExtendedReq {
+    pub hdr: MailboxReqHeader,
+    pub certify_key_req: [u8; CertifyKeyExtendedReq::CERTIFY_KEY_REQ_SIZE],
+    pub flags: u32,
```

Comment:
nit: Will probably make the code more readable if you use a bitflags type here. Shouldn't change the underlying type, so I assume you can still use repr C.

Reply:
Done